app/views/internal/feedback_messages/index.html.erb

```diff
@@ -80,7 +80,7 @@
     <div id="vomitReactionsBodyContainer" class="collapse hide" aria-labelledby="vomitReactionsHeader" data-parent="#vomitReactions">
       <div class="card-body" style="overflow: scroll; max-height: 500px;">
         <% @vomits.each do |reaction| %>
-          <% next if (reaction.reactable_type == "Article" && !reaction.reactable.published) || (reaction.reactable_type == "User" && reaction.reactable.banned) %>
+          <% next if (reaction.reactable_type == "Article" && !reaction.reactable.published) || (reaction.reactable_type == "User" && reaction.reactable.username.starts_with?("spam_")) %>
```

Comment:
Could we add this as a method on the user model?

Side-note: This verbiage is super confusing.

Reply:
Re: verbiage, yeah we've been trying to move toward suspended instead, although it's not yet changed in the code.